uber/models.py

```diff
@@ -401,6 +401,11 @@ def apply(self, params, *, bools=(), checkgroups=(), restricted=True, ignore_csr
                             value = None
                         else:
                             value = float(value)
+                    elif isinstance(column.type, Numeric):
+                        if value == '':
+                            value = None
+                        elif value.endswith('.0'):
+                            value = int(float(value))
```

Comment:
Haha, Numeric is fine :) I had no idea we supported fractional tables. But it really makes sense when you think about somebody that might only need half a table!